### PR TITLE
Add a new function to run cwp without exiting a process

### PR DESCRIPTION
**Pull request by jxr98**
**Make sure that you've checked the boxes below before you submit PR:**

### Always

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side) and not your master branch!
- [x] Written well with PR title, we generate the [release notes](https://github.com/jenkins-zh/jenkins-cli/releases) base on that

### For the bug fixes or features only

- [ ] [Quality Gate Passed](https://sonarcloud.io/dashboard?id=jenkins-zh_jenkins-cli). Change this URL to your PR.
- [ ] The coverage is xxx on the new lines
- [ ] I've tested it by manual in the following platform
  - [ ] MacOS
  - [x] Linux
  - [ ] Windows
- [ ] Unit Test covered
- [ ] e2e Test covered

### CODE REVIEW

**Comment by jxr98** [comment]
I want to use the cwpOptions.Run() method in other go file, but this function uses syscall.Exec() which ends a process and the following code are not executing. So I creates a new function called RunWithoutProcessExits() and the logic is the same basically, while using exec.Command() which doesn't exiting a process instead of syscall.Exec().

File: app/cmd/cwp.go [hunk 1]
```diff
@@ -174,6 +175,68 @@ func (o *CWPOptions) Run(cmd *cobra.Command, args []string) (err error) {
 	return
 }
 
+//RunWithoutProcessExits has the same logic with function Run and the difference between them is that RunWithoutProcessExits uses exec.Command() instead of syscall.Exec()
+//which causes the process to exit without executing code after cwp.Run()
+func (o *CWPOptions) RunWithoutProcessExits(cmd *cobra.Command, args []string) (err error) {
```
Comment (LinuxSuRen) [inline app/cmd/cwp.go:180]: The comment should be like: `// RunWithoutProcessExits xxxx`

You need to add a space.

File: app/cmd/cwp.go [hunk 2]
```diff
@@ -174,6 +175,68 @@ func (o *CWPOptions) Run(cmd *cobra.Command, args []string) (err error) {
...
+			return
+		}
+	} else if err != nil {
+		return
+	}
+
+	_, err = util.LookPath("java", o.LookPathContext)
+	if err == nil {
+		// env := os.Environ()
+
```
Comment (LinuxSuRen) [inline app/cmd/cwp.go:199]: Please remove the unused code lines.

**Review by LinuxSuRen — CHANGES_REQUESTED** [review]
Thanks for your contribution. There are a few small problems you might need to take a look at them.

File: app/cmd/cwp.go (hunk 1 above)
Comment (jxr98) [reply app/cmd/cwp.go:180]: Thx. It's done.

**Comment by jxr98** [comment]
The e2e Test/ Build ubuntu failed, but I don't know why. Could someone give me a hint on it. That would be appreciated.

**Comment by jxr98** [comment]
Hello Rick! This pr is needed to be merged first. Thanks for your reviewing.
An example to use this new added function is as follows:
 ```
var rootCmd = &cobra.Command{
  Use:   "sth",
  Short: "sth",
  RunE: func(cmd *cobra.Command, args []string)(err error) {
    var cwpOptions CWPOptions
    cwpOptions.ConfigPath = pathToYamlFile
    err =cwpOptions.RunWithoutProcessExits(cmd, args)
   return err
  },
}
```
But this is not a complete file and more information about cobra is available [here](https://github.com/spf13/cobra/blob/master/user_guide.md).

**Review by LinuxSuRen — APPROVED** [review]

**Pull request merged**